jdi-light/src/main/java/com/epam/jdi/light/settings/TestProperties.java

```diff
@@ -0,0 +1,263 @@
+    testsProperties.put("assert.type",
+        new Pair<>(SoftAssert::setAssertType,doNothing));
+    testsProperties.put("click.type",
+        new Pair<>(p -> ELEMENT.clickType = getClickType(p),doNothing));
+    testsProperties.put("text.type",
+        new Pair<>(p -> ELEMENT.getTextType = getTextType(p),doNothing));
+    testsProperties.put("set.text.type",
+        new Pair<>(p -> ELEMENT.setTextType = getSetTextType(p),doNothing));
+//     RemoteWebDriver properties
+    testsProperties.put("remote.type",
+        new Pair<>(p -> DRIVER.remoteUrl = getRemoteUrl(p),doNothing));
+    testsProperties.put("driver.remote.url",
+        new Pair<>(p -> DRIVER.remoteUrl = p,doNothing));
+    testsProperties.put("log.level",
+        new Pair<>(p -> LOGS.logLevel = parseLogLevel(p), () -> logger.setLogLevel(LOGS.logLevel)));
+    testsProperties.put("allure.steps",
+        new Pair<>(p -> LOGS.writeToAllure = parseBoolean(p),doNothing));
+    testsProperties.put("smart.locators",
+        new Pair<>(p -> ELEMENT.smartTemplate = p.split(";")[0],doNothing));
+    testsProperties.put("smart.locators.toName",
+        new Pair<>(p -> ELEMENT.smartName = getSmartSearchFunc(p),doNothing));
+    testsProperties.put("smart.search",
+        new Pair<>(p -> ELEMENT.useSmartSearch = getSmartSearchUse(p),doNothing));
+    testsProperties.put("headless",
+        new Pair<>(p -> DRIVER.capabilities.common.put("headless", p),doNothing));
+  }
+
+  private static Strategies getStrategy(String prop) {
+    String strategy = prop.trim().toLowerCase();
+    switch (strategy) {
+      case "jdi":
+        return JDI;
+      case "selenium":
+        return SELENIUM;
+      default:
+        return JDI;
+    }
+  }
+
+  private static List<Strategy> getLoggerStrategy(String strategy) {
+    if (isBlank(strategy))
+      return new ArrayList<>();
+    List<com.epam.jdi.light.logger.Strategy> strategies = new ArrayList<>();
+    try {
+      String[] split = strategy.split(";");
+      strategies = LinqUtils.map(split, s -> parseStrategy(s.trim()));
+    } catch (Exception ignore) { }
+    return strategies;
+  }
+
+  private static void setSearchStrategy(String p) {
+    p = p.toLowerCase();
+    if (p.equals("soft"))
+      p = "any, multiple";
+    if (p.equals("strict"))
+      p = "visible, single";
+    if (p.split(",").length == 2) {
+      List<String> params = asList(p.split(","));
+      if (params.contains("visible") || params.contains("displayed"))
+        onlyVisible();
+      if (params.contains("any") || params.contains("all"))
+        noValidation();
+      if (params.contains("enabled"))
+        visibleEnabled();
+      if (params.contains("inview"))
+        inView();
+      if (params.contains("single"))
+        STRICT_SEARCH = true;
+      if (params.contains("multiple"))
+        STRICT_SEARCH = false;
+    }
+  }
+
+  private static PageLoadStrategy getPageLoadStrategy(String strategy) {
+    switch (strategy.toLowerCase()) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Switch statements should have a default label](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5524501)